### PR TITLE
Add macos tests

### PR DESCRIPTION
**Pull request by Luni-4**
Test `pyffms2` on `macos`

### CODE REVIEW

File: .github/workflows/pyffms2.yml [hunk 1]
```diff
@@ -6,39 +6,54 @@ jobs:
...
+    - name: Install FFmpeg on Ubuntu
+      if: matrix.os == 'ubuntu-latest'
       run: |
         sudo add-apt-repository ppa:jonathonf/ffmpeg-4
         sudo apt-get install libavcodec-dev libavformat-dev libavdevice-dev
 
+    - name: Install FFmpeg on MacOS
+      if: matrix.os == 'macos-latest'
+      run: |
+        brew install ffmpeg
```
Comment (rr-) [inline .github/workflows/pyffms2.yml]: Is it possible to merge these dependency steps into one, or one for each OS? What's the best practice for more robust `if` statements?

Comment (Luni-4) [reply .github/workflows/pyffms2.yml]: Something like this:

```
- name: Install FFmpeg
  run: |
       if: matrix.os == 'macos-latest'
           brew install ffmpeg
       elif: matrix.os == 'ubuntu-latest'
           sudo ..
```
If yes, nope you can't do it. The `if` structure acts only on the entire step, you can't specify it inside the `run` tag.

Comment (Luni-4) [reply .github/workflows/pyffms2.yml]: We could use the shell (the `uname` command) though, but I don't think it would be the right approach.

Comment (rr-) [reply .github/workflows/pyffms2.yml]: agreed

Comment (Luni-4) [reply .github/workflows/pyffms2.yml]: On what? Do you want the shell approach or merging this PR?

**Pull request merged**